amundsen_application/static/js/components/common/TagsList/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import './styles.scss';
+
+import AppConfig from 'config/config';
+import LoadingSpinner from 'components/common/LoadingSpinner';
+import TagInfo from 'components/Tags/TagInfo';
+import { Tag } from 'interfaces';
+
+import { GlobalState } from 'ducks/rootReducer';
+import { getAllTags } from 'ducks/allTags/reducer';
+import { GetAllTagsRequest } from 'ducks/allTags/types';
+
+export interface StateFromProps {
+  allTags: Tag[];
+  isLoading: boolean;
+}
+
+export interface DispatchFromProps {
+  getAllTags: () => GetAllTagsRequest;
+}
+
+interface TagsListState {
+  curatedTags: Tag[];
+  otherTags: Tag[];
+  isLoading: boolean;
+}
+
+export type TagsListProps = StateFromProps & DispatchFromProps;
+
+export class TagsList extends React.Component<TagsListProps, TagsListState> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      curatedTags: [],
+      otherTags: [],
+      isLoading: this.props.isLoading,
```

Comment:
I know I wrote this code originally, but we can remove `isLoading` from `this.state` and just use `this.props.isLoading` instead.

Reply:
+1. Don't forget to remove it from the `TagsListState` interface as well